app/views/home/maintainers.md

```diff
@@ -0,0 +1,45 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that identify as a "maintainer" regularly contribute to an open source project.
```

Comment:
I think this is pretty valuable to people that haven't realized that they should be optimizing their project for new contributors.